clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -11093,15 +11117,14 @@ void LinkerWrapper::ConstructJob(Compilation &C, const JobAction &JA,
     types::ID OutputType = TargetTriple.isSPIROrSPIRV() || IsSYCLNativeCPU
                                ? types::TY_Tempfiletable
                                : types::TY_LLVM_BC;
-    // TODO: Items like native_cpu and Specialization Constants behaviors are
-    // dependent on each toolchain.  Passing these along as 'general settings'
-    // for the clang-linker-wrapper causes for potential inconsistencies and
-    // would need to handled more at the device linking level.
     bool SpecConsts = TargetTriple.isSPIROrSPIRV();
-    getOtherSYCLPostLinkOpts(getToolChain(), JA, Args, PostLinkArgs, SpecConsts,
-                             OutputType);
+    getOtherSYCLPostLinkOpts(getToolChain(), JA, Args, PostLinkArgs);
```

Comment:
Is there a category of options here we can use to be more specific about what options are set here rather than `Other`?